homeassistant/components/simplisafe/__init__.py

```diff
@@ -307,10 +302,10 @@ async def async_reload_entry(hass, config_entry):
 class SimpliSafe:
     """Define a SimpliSafe data object."""
 
-    def __init__(self, hass, api, config_entry):
+    def __init__(self, hass, config_entry, api, get_api_coro):
```

Comment:
A coro is an abbreviation for coroutine. A coroutine is the return value of a coroutine function. 

I would name this `async_get_api`